vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewState.kt

```diff
@@ -47,10 +51,5 @@ data class CreateRoomViewState(
     fun isEmpty() = avatarUri == null
             && roomName.isEmpty()
             && roomTopic.isEmpty()
-            && (roomVisibilityType as? RoomVisibilityType.Public)?.aliasLocalPart?.isEmpty().orTrue()
-
-    sealed class RoomVisibilityType {
-        object Private : RoomVisibilityType()
-        data class Public(val aliasLocalPart: String) : RoomVisibilityType()
```

Comment:
I'm a bit sad to see this model disappearing. It was clearer that alias is mandatory for public room.
Can it be restored, and add a class for restricted?